src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj

```diff
@@ -86,5 +86,6 @@
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
><PackageReference Include="Newtonsoft.Json" Version="9.0.1" /> [](start = 4, length = 62)

is this still needed?

Reply:
Yes. There is still code using Newtonsoft. Which will be changed in upcoming PRs.